src/status_im2/contexts/quo_preview/inputs/title_input.cljs

```diff
@@ -0,0 +1,54 @@
+(ns status-im2.contexts.quo-preview.inputs.title-input
+  (:require [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
+            [quo2.core :as quo]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label "disabled?"
+    :key   :disabled?
+    :type  :boolean}
+   {:label "blur?"
+    :key   :blur?
+    :type  :boolean}
+  ])
+
+(defn cool-preview
+  []
+  (let [state          (reagent/atom {:blur?     false
+                                      :disabled? false})
+        value          (reagent/atom "")
+        max-length     24
+        on-change-text (fn [v]
+                         (reset! value v)
+                         (reagent/flush)
```

Comment:
without reagent/flush the text flickers on input..

researched this here:
https://github.com/reagent-project/reagent/issues/485#issuecomment-613551238


Should we take a different approach for input components, I see quo text inputs are quite complicated although perhaps this is necessary for a smooth input effect. 

In any case we should probably have the base functionality available for all input components and then we can stylise the ui as needed for the various components.

Would appreciate some input (🤓) on this

Reply:
`:default-value` should be used, we shouldn't pass any value to the input

Reply:
how do you mean? surely the on change has to pass the updated value to the `reset` function ?

Reply:
https://gist.github.com/ilmotta/9753c003359bda9be2afd85d4e4a78d9

@J-Son89, see the gist I created now. I tested now and it works correctly. It follows some of the suggestions by @flexsurfer